tests/test_mc_run.py

```
@@ -1,73 +1,57 @@
-@pytest.mark.parametrize('row, expected,_', good_params, ids=range(len(good_params)))
-def test__row_parser(row, expected, _, tweak_factory):
+@pytest.mark.parametrize(('row', 'expected', '_'), good_params)
+def test__row_parser(row: str, expected: RowData, _: object, tweak_factory: TweakFactory) -> None:
     assert tweak_factory.row_parser(0, row=row) == expected
 
 
-@pytest.mark.parametrize('row', fail_examples, ids=ids)
-def test__row_parser_fail(row, tweak_factory):
+@pytest.mark.parametrize('row', fail_examples)
+def test__row_parser_fail(row: str, tweak_factory: TweakFactory) -> None:
     with pytest.raises(ValueError):
```

Ruff (0.14.5)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)